doc/data/messages/b/boolean-datetime/bad.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
We'd need an example of bad code for this message

Reply:
The error doesn't happen unless you use Python version 3.5

Reply:
Use py-version=3.5 like us f-string-used- ... in version that does not support it.

Reply:
Ok, I stopped changing boolean-datetime